packages/contracts-bedrock/test/actors/FaultDisputeActors.sol

```diff
@@ -389,10 +405,14 @@ contract HonestDisputeActor is DisputeActor {
                 });
             }
 
-            (bool innerSuccess,) = address(GAME).call(localMove.data);
+            (bool innerSuccess,) = address(GAME).call{ value: localMove.value }(localMove.data);
             assembly {
                 success_ := and(success_, innerSuccess)
             }
         }
     }
+
+    fallback() external payable { }
+
+    receive() external payable { }
```

Comment:
Since both are a no-op `receive` zero-calldata calls will be caught by `fallback` anyways, so only need to define the `fallback` selector here.
```suggestion
    fallback() external payable { }
```

Reply:
This produces a warning, solc 🫠